app/assets/v2/js/pages/bounty_details.js

```diff
@@ -905,6 +905,10 @@ var build_detail_page = function(result) {
   });
 
   $('#bounty_details #issue_description code').parent().addClass('code-snippet');
+  if (result['value_in_usdt_now'] != 'undefined' && result['value_in_usdt_now'] <= 1) {
+    $('#value_in_usdt_wrapper').hide();
+    $('.token').hide();
+  }
```

Comment:
Do we need this ? 
we could do these checks within results.html itself using template `if`

Reply:
@danlipert ^ in case you didn't see it

Reply:
@danlipert this can be removed the changes in the template are doing this check already

Reply:
Ideally, this should be done only in live ! we might need to revisit it